pxd.c

```diff
@@ -1770,6 +1779,7 @@ static struct attribute *pxd_attrs[] = {
 	&dev_attr_fastpath.attr,
 	&dev_attr_mode.attr,
 	&dev_attr_debug.attr,
+	&dev_attr_inprogress.attr,
```

Comment:
Ok, but isn't this same as device "inflight" count?

Reply:
yes should be same, except for the window wherein IO is considered active. 
device inflight is controlled from within the px-fuse driver from places where it calls start_io_acct and end_io_acct.
The exported count is wholly private to the driver.